kubernetes_pod_security_policy host_ports crash

### Terraform Version and Provider Version
```
Terraform v0.12.29
+ provider.kubernetes v1.11.4
```

### Affected Resource(s)

- kubernetes_pod_security_policy

### Terraform Configuration Files
```hcl
resource "kubernetes_pod_security_policy" "this" {
  metadata {
    name = "restricted"
  }
  spec {
    privileged                 = false
    allow_privilege_escalation = false
    allowed_capabilities       = []

    volumes = [
      "configMap",
      "emptyDir",
      "projected",
      "secret",
      "downwardAPI",
      "persistentVolumeClaim",
    ]

    host_network = false
    host_ipc     = false
    host_pid     = false

    host_ports {
      min = 0
      max = 65535
    }

    run_as_user {
      rule = "MustRunAsNonRoot"
    }

    se_linux {
      rule = "RunAsAny"
    }

    supplemental_groups {
      rule = "MustRunAs"
      range {
        min = 1
        max = 65535
      }
    }

    fs_group {
      rule = "MustRunAs"
      range {
        min = 1
        max = 65535
      }
    }

    read_only_root_filesystem = true
  }
}
```

### Debug Output
https://gist.github.com/Filirom1/099cc167516002aa0eda9cacacf926c5

### Panic Output
https://gist.github.com/Filirom1/099cc167516002aa0eda9cacacf926c5

### Expected Behavior
```
kubernetes_pod_security_policy.this: Creating...
kubernetes_pod_security_policy.this: Creation complete after 3s [id=restricted]

Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
```

### Actual Behavior
```
kubernetes_pod_security_policy.this: Creating...

Error: rpc error: code = Canceled desc = context canceled


panic: interface conversion: interface {} is int, not int32
2020-07-24T12:30:11.788+0200 [DEBUG] plugin.terraform-provider-kubernetes_v1.11.4_x4:
2020-07-24T12:30:11.788+0200 [DEBUG] plugin.terraform-provider-kubernetes_v1.11.4_x4: goroutine 26 [running]:
2020-07-24T12:30:11.788+0200 [DEBUG] plugin.terraform-provider-kubernetes_v1.11.4_x4: github.com/hashicorp/terraform-provider-kubernetes/kubernetes.expandHostPortRangeSlice(0xc000729460, 0x1, 0x1, 0xa, 0xc0006a7cd8, 0xc000345970)
2020-07-24T12:30:11.788+0200 [DEBUG] plugin.terraform-provider-kubernetes_v1.11.4_x4:   /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-kubernetes/kubernetes/structure_pod_security_policy_spec.go:386 +0x1a6
...
```

### Steps to Reproduce

1. `terraform apply` 

### Important Factoids

Nothing

### References
- https://github.com/hashicorp/terraform-provider-kubernetes/pull/861

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

--- a/kubernetes/resource_kubernetes_pod_security_policy_test.go
+++ b/kubernetes/resource_kubernetes_pod_security_policy_test.go
@@ -278,6 +278,11 @@ resource "kubernetes_pod_security_policy" "test" {
 		max = 65535
 	  }
     }
+
+    host_ports {
+      min = 0
+      max = 65535
+    }
     
     read_only_root_filesystem = true
   }

--- a/kubernetes/structure_pod_security_policy_spec.go
+++ b/kubernetes/structure_pod_security_policy_spec.go
@@ -149,8 +149,8 @@ func flattenHostPortRangeSlice(in []v1beta1.HostPortRange) []interface{} {
 
 	for k, v := range in {
 		result[k] = map[string]interface{}{
-			"min": v.Min,
-			"max": v.Max,
+			"min": int(v.Min),
+			"max": int(v.Max),
 		}
 	}
 
@@ -383,8 +383,8 @@ func expandHostPortRangeSlice(in []interface{}) []v1beta1.HostPortRange {
 	for k, v := range in {
 		if m, ok := v.(map[string]interface{}); ok {
 			result[k] = v1beta1.HostPortRange{
-				Min: m["min"].(int32),
-				Max: m["max"].(int32),
+				Min: int32(m["min"].(int)),
+				Max: int32(m["max"].(int)),
 			}
 		}
 	}

